projects/coredns/coredns/Makefile

```diff
@@ -15,7 +15,7 @@ AWS_REGION?=us-west-2
 AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 
 MAKE_ROOT=$(shell cd "$(shell dirname "${BASH_SOURCE[0]}")" && pwd -P)
-BASE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+BASE_REPO?="public.ecr.aws/h1r8a7l5"
```

Comment:
this is our development build. maybe we should register a name for it so people know what they are getting. Another alternative would be move the base image build into this repository which would solve a lot of problems

Reply:
+1 on registering a name, but can we register a name on an unlisted repo? Also do you think we can push this image to the verified public ECR repo and consume from there, instead of using dev build?